.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: Run codebase checks and tests
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    services:
+      db:
```

Comment:
I think you should had 
```
db:
 env:
    POSTGRES_USER: postgres
    POSTGRES_PASSWORD: postgres
    POSTGRES_DB: postgres
```

Or you gonna have the following error : `Failed to initialize, db service is unhealthy.`

Reply:
It works with specific environement variables, but it might be related to the healthcheck options.

https://github.com/mirego/elixir-boilerplate/pull/91/files#diff-60fe268f71ae7f8409aff40d0a3e51d8R20